circle.yml

```diff
@@ -0,0 +1,19 @@
+general:
+  artifacts:
+    - "xcodebuild.log"
+    - "build/Logs"
```

Comment:
build/Logs are the xcactivity logs, we'd also need to save the `PlayerKit_Example-PlayerKit-Example.log` here too